### PR TITLE
[New Model]: Renamed Model: Secondary Material Content

### LINKED ISSUE #261: [Model Update]: Material For Recycling
## Update Reason
 Current data model is insufficient because it doesn't describe general material properties and detailed secondary material content properties. The updated model consolidates input from Material for Homologation, Material for Recycling and Secondary Material Content and proposes one data model. 
-> Current existing data model: Material for recycling
--> use case Circular Economy, Circularity Kit - Secondary Material Content Feature 
--> model update has been discussed with all relevant stakeholders 
## MS1 Criteria

- [ ] The model that should be updated exists
- [ ] The referenced use case exists
- [ ] The potential updates are discussed with all stakeholders
- [ ] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
- [ ] The potential update will be either 
  - [ ] backward compatible or 
  - [ ] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed

### PR DESCRIPTION
**Pull request by catroest**
## Description

SMC Model includes new secondary characteristic attributes.
Renamed from Material for Recycling.

Closes #261 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the BAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar bamm-cli.jar -i \<path-to-aspect-model\> -v ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.3.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the BAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by catroest** [comment]
@bs-jokri PR on behalf of Sabine Wichmann and Johannes Schramm

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.secondary_material_content:1.0.0#> .
+@prefix ext-part: <urn:samm:io.catenax.serial_part:1.0.1#> .
```
[the remark this answers is not in the document]
Comment (bs-jokri) [reply io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:28]: PR #291 is created, which updates SerialPart to v200 Please update the reference to 
<urn:samm:io.catenax.serial_part:2.0.0#> .
to use the updated model.

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+
+:SecondaryMaterialContent a samm:Aspect ;
+   samm:preferredName "Secondary Material Content"@en ;
+   samm:description "This model refers to a part containing one or more different material. It references the share on primary and secondary material and relevant details."@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:32]: materials?

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+   samm:properties ( :secondaryMaterialContent :supplierID :orderNumber ext-part:manufacturerPartId ext-part:customerPartId ext-part:nameAtManufacturer ext-part:nameAtCustomer ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:secondaryMaterialContent a samm:Property ;
+   samm:preferredName "secondary Material Content"@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:38]: Start uppercase (secondary -> Secondary)

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+   samm:description "List of relevant materials which are built in this part and details on the share of primary and secondary material."@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:39]: material -> materials?

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+   samm:characteristic :SecondaryMaterialContentMaterials .
+
+:supplierID a samm:Property ;
+   samm:preferredName "supplier ID"@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:43]: uppercase

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 6]
```diff
@@ -0,0 +1,417 @@
...
+   samm:description "Suppliers individual business ID within the Catena-X network."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "BPN1234567890ZZ" .
+
+:orderNumber a samm:Property ;
+   samm:preferredName "order Number"@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:49]: uppercase -> please check all further preferredNames if they start uppercase

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 7]
```diff
@@ -0,0 +1,417 @@
...
+   samm:description "Order number as provided by supplier."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "856496" .
+
+:SecondaryMaterialContentMaterials a samm-c:Set ;
+   samm:preferredName "Secondary Material Content Materials"@en ;
+   samm:description "List of relevant materials which are built in this part and details on the share of primary and secondary material."@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:56]: check for singular and plural use of material. I am also not sure, whether this is some domain specific use of the term but for me it sounds like primary and secondary are two materials thus would opt for materials. But maybe I do not get that right

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 8]
```diff
@@ -0,0 +1,417 @@
...
+   samm:dataType :SecondaryMaterialContentMaterialsEntity .
+
+:SecondaryMaterialContentMaterialsEntity a samm:Entity ;
+   samm:preferredName "Secondary Material Content Materials Entity"@en ;
+   samm:description "List of relevant materials which are built in this part and details on the share of primary and secondary material."@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:61]: materials, which

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 9]
```diff
@@ -0,0 +1,417 @@
...
+   samm:properties ( :materialNameAtSupplier :materialClass :materialNameStandardized :unitOfMeasure :bioBased :inorganic :certificate ) .
+
+:materialNameAtSupplier a samm:Property ;
+   samm:preferredName "material name at supplier"@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:65]: Material Name at Supplier reason to write uppercase is the use of preferredName as proper noun

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 10]
```diff
@@ -0,0 +1,417 @@
...
+   samm:description "Material name as provided by supplier."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Opatex" .
+
+:materialClass a samm:Property ;
+   samm:preferredName "material Class"@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:71]: Material Class

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 11]
```diff
@@ -0,0 +1,417 @@
...
+   samm:description "Material class according to VDA standard 231-106."@en ;
+   samm:see <https://www.mdsystem.com/imdsnt/startpage/index.jsp> ;
+   samm:characteristic :MaterialClass ;
+   samm:exampleValue "5.1" .
+
+:materialNameStandardized a samm:Property ;
+   samm:preferredName "material name standardized"@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:78]: maybe Standardized Material Name

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 12]
```diff
@@ -0,0 +1,417 @@
...
+   samm:description "Material name according to standard."@en ;
+   samm:characteristic :MaterialNameStandardized .
+
+:unitOfMeasure a samm:Property ;
+   samm:preferredName "unit of measure"@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:83]: please check the remaining preferred names for uppercase writing and correct accordingly

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 13]
```diff
@@ -0,0 +1,417 @@
...
+   samm:properties ( :secondaryInorganic :primaryInorganic ) .
+
+:CertificateEntity a samm:Entity ;
+   samm:preferredName "Certificate Entity"@en ;
+   samm:description "If available, the name of a certificate verifying the recycling content can be included."@en ;
+   samm:properties ( :certificateName :certificateLink ) .
+
+:referencedStandard a samm:Property ;
+   samm:preferredName "referenced Standard"@en ;
+   samm:description "Norm authority who governs norm."@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:160]: which governs ...

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 14]
```diff
@@ -0,0 +1,417 @@
...
+   samm:characteristic :ReferencedStandard ;
+   samm:exampleValue "ISO" .
+
+:referencedStandardID a samm:Property ;
+   samm:preferredName "referenced Standard ID"@en ;
+   samm:description "Norm code of related naming norm."@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:166]: don't get the description. Maybe something like
Id of standard used by standardization body?

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 15]
```diff
@@ -0,0 +1,417 @@
...
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "1043" .
+
+:materialNameStandardizedValue a samm:Property ;
+   samm:preferredName "material Name Standardized Value"@en ;
+   samm:description "Name of the material according to the norming authority and the referenced ID of the norm."@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:172]: is norming authority the right term? I have heard the term standardization body?

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 16]
```diff
@@ -0,0 +1,417 @@
...
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "PP-TD10" .
+
+:unitOfMeasureKey a samm:Property ;
+   samm:preferredName "unit of Measure Key"@en ;
+   samm:description "Unit of measure in which the mass of the material is being measured."@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:178]: is measured

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 17]
```diff
@@ -0,0 +1,417 @@
...
+   samm:exampleValue "Redcert" .
+
+:certificateLink a samm:Property ;
+   samm:preferredName "certificateLink"@en ;
+   samm:description "The certificate can be referenced for example as a PDF file."@en ;
+   samm:characteristic samm-c:ResourcePath .
+
+:ReferencedStandard a samm-c:Enumeration ;
+   samm:preferredName "Referenced Standard"@en ;
+   samm:description "Norm authority who governs norm."@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:227]: which

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 18]
```diff
@@ -0,0 +1,417 @@
...
+   samm:dataType :SecondaryInorganicEntity .
+
+:PrimaryInorganic a samm:Characteristic ;
+   samm:preferredName "Primary Inorganic"@en ;
+   samm:description "Primary materials are raw materials that are used for the first time during the planned process."@en ;
+   samm:dataType :PrimaryInorganicEntity .
+
+:UnitReference a samm:Characteristic ;
+   samm:preferredName "Unit Reference"@en ;
+   samm:description "Describes a Property containing a reference to one of the units in the Unit Catalog."@en ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:268]: Property -> property

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 19]
```diff
@@ -0,0 +1,417 @@
...
+   samm:see <https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/appendix/unitcatalog.html#unit-catalog-units> ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:269]: replace by current reference 
https://eclipse-esmf.github.io/samm-specification/2.0.0/appendix/unitcatalog.html

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 20]
```diff
@@ -0,0 +1,417 @@
...
+   samm:dataType samm:curie .
+
+:AllowMassUnit a samm-c:RegularExpressionConstraint ;
+   samm:description "Only allow mass related Units"@en ;
+   samm:see <https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/appendix/unitcatalog.html#unit-catalog-units> ;
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:274]: same here, replace with updated url

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 21]
```diff
@@ -0,0 +1,417 @@
...
+:PrimaryInorganic a samm:Characteristic ;
+   samm:preferredName "Primary Inorganic"@en ;
+   samm:description "Primary materials are raw materials that are used for the first time during the planned process."@en ;
+   samm:dataType :PrimaryInorganicEntity .
+
+:UnitReference a samm:Characteristic ;
+   samm:preferredName "Unit Reference"@en ;
+   samm:description "Describes a Property containing a reference to one of the units in the Unit Catalog."@en ;
+   samm:see <https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/appendix/unitcatalog.html#unit-catalog-units> ;
+   samm:dataType samm:curie .
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:270]: if you use curie as data type to reference a measurement unit, be aware of that it has the structure

unit:<measurement_unit> -> The regex, below will not work for that. So probably you might use xsd:string instead if you want to stick to the regex.

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl [hunk 22]
```diff
@@ -0,0 +1,417 @@
...
+   samm:preferredName "is Mass Balanced"@en ;
+   samm:description "Information whether mass-balancing was applied: Mass balance is one of several well-known chain of custody approaches that can be used to track the flow of materials through the value chain. The total quantity of the content in question is tracked through the production system and the allocation of this content is made to the end products based on verifiable accounting."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:percentageOfMaterialWeight a samm:Property ;
+   samm:preferredName "percentage Of Material Weight"@en ;
+   samm:description "Share of the full material weight in percentage."@en ;
+   samm:characteristic :PercentageOfMaterialWeightTrait ;
+   samm:exampleValue "12"^^xsd:float .
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:307]: to make it clear that it is a floating point value I would write 12.0

File: io.catenax.secondary_material_content/RELEASE_NOTES.md [hunk 23]
```diff
@@ -0,0 +1,15 @@
+# Changelog
+All notable changes to this model will be documented in this file.
+
+## [Unreleased]
+
+## [1.0.0] - 2023-08-28
```
Comment (bs-jokri) [inline io.catenax.secondary_material_content/RELEASE_NOTES.md:6]: replace with date of MS3 approval

**Review by bs-jokri — CHANGES_REQUESTED** [review]
please check my comments

**Comment by bs-jokri** [comment]
The checks for MS2 where done with the updated model provided with PR https://github.com/eclipse-tractusx/sldt-semantic-models/pull/291

**Comment by bs-jokri** [comment]
@catroest @johannsvarela there are still open review comments, who will resolve them? Otherwise we cannot assign MS2

File: io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl (hunk 15 above)
Comment (agg3fe) [reply io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl:166]: Had a talk with respective team, yes this 'norming authority' is correct.

**Review by agg3fe — APPROVED** [review]
LGTM

**Review by bs-jokri — APPROVED** [review]

**Comment by catroest** [comment]
@bs-jokri eventually followed by a deprecation request of the old one

**Comment by bs-jokri** [comment]
> @bs-jokri eventually followed by a deprecation request of the old one

@catroest could you link the deprecation request here?

**Comment by catroest** [comment]
#395 old one will be deprecated

**Pull request merged**